opennms-doc/guide-admin/src/asciidoc/text/snmp/snmp-encryption.adoc

```diff
@@ -0,0 +1,19 @@
+
+=== Encrypting SNMP Credentials
+
+SNMP credentials like readCommunity/WriteCommunity, authPassPhrase/privPassPhrase can be saved on the file system in encrypted format.
+
+==== Enabling encrypting on SNMP Config
+
+Enable encryption on SNMP config by setting following system property.
+[source, sh]
+----
+echo 'org.opennms.snmp.encryption.enabled=true' >> "$OPENNMS_HOME/etc/opennms.properties.d/snmp-encryption.properties"
+----
+
+SNMP config that is modified on User Interface will be encrypted directly on file system.
+Config that is updated on the file system needs to be saved with karaf command `opennms-snmp:save-config` to get encrypted.
```

Comment:
Would be good to document how the key used to encrypt/decrypt the attributes is managed, the fact that the key is stored in SCV and what alias the key is stored under.